portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** An item on a comment section */
+public final class Comment {
+
+  private final long id;
+  private final String username;
+  private final String commentText;
+  private final long timestamp;
+
+  public Comment(long id, String username, String commentText, long timestamp) {
```

Comment:
This class and this function should have javadoc comments.
https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required

Reply:
Can you also add the block tags for the parameters: https://google.github.io/styleguide/javaguide.html#s7.1.3-javadoc-block-tags

Reply:
Thanks @lorikeet-s, I just made the change. I am not sure whether this is how it was supposed to be done, so let me know if it needs to be modified again

Reply:
looks good things, for other functions you should have a return, but you don't normally add one for constructors I don't think.